agent/ui_endpoint.go

```diff
@@ -343,6 +343,20 @@ RPC:
 		}
 		upstreamResp = append(upstreamResp, &sum)
 	}
+	for k, v := range out.ServiceTopology.UpstreamSources {
+		if v == structs.TopologySourceRoutingConfig {
+			sn := structs.ServiceNameFromString(k)
+			sum := ServiceTopologySummary{
+				ServiceSummary: ServiceSummary{
+					Name:           sn.Name,
+					EnterpriseMeta: sn.EnterpriseMeta,
+				},
```

Comment:
How this looks is WIP; will need to coordinate with FE to expose the fields they need to navigate to a routing-config view.

Reply:
Looks like I'll need to provide datacenter as well, which is going to be tricky!